github/issues_test.go

```diff
@@ -149,7 +150,7 @@ func TestIssuesService_Get(t *testing.T) {
 		fmt.Fprint(w, `{"number":1, "labels": [{"url": "u", "name": "n", "color": "c"}]}`)
 	})
 
-	issue, _, err := client.Issues.Get("o", "r", 1)
+	issue, _, err := client.Issues.Get(context.Background(), "o", "r", 1)
```

Comment:
just for brevity, we might want to have a package-level:

```go
var ctx = context.Background()
```

(or is that discouraged somewhere?)